inc/config.class.php

```diff
@@ -466,7 +466,7 @@ function changestatus() {
       echo "</tr>";
 
       echo "<tr class='tab_bg_1'>";
-      echo "<td>".__('Network equpment')."</td>";
+      echo "<td>".__('Network equipment')."</td>";
```

Comment:
This string does not seems to exists in core, so:
```suggestion
      echo "<td>".__('Network equipment', 'fusioninventory')."</td>";
```